coderd/workspacestats/wstest/batcher.go

```diff
@@ -0,0 +1,38 @@
+package wstest
```

Comment:
IMO it should be `workspacestatstest` or at least `statstest`. `wstest` sounds like "workspace test"

Reply:
originally had `workspacestatstest` and thought I'd be told it's too verbose lol, 👍